app/controllers/api/v1/admin/display_users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Admin::DisplayUsersController < Api::V1::ApplicationController
+  include AdminAuthorization
+
+  def index
+    @users = User.where(role: :user)
+    render json: @users, each_serializer: UserSerializer
```

Comment:
you can make this a 1-liner: `render json: User.where(role: :user)`. Optionally, you can also add `, each_serializer: UserSerializer`, but it should use the correct serializer anyway